.github/workflows/movieLens-py37.yml

```diff
@@ -8,7 +8,8 @@ on:
     branches-ignore:
       - master
 jobs:
-  testbox:
+  build:
+    name: Test on ${{ matrix.python-version }}
     runs-on: ubuntu-latest
```

Comment:
Yes. If no image version is provided, the latest version is pulled, I've tagged and pushed the gdmix-dev image so that the integration test succeeded.